small-talk/app/components/SidebarButton.js

```diff
@@ -16,16 +16,12 @@ const SidebarButton = ({ redirect, defaultImg, hoverImg, altText }) => {
     const onMouseLeave = () => setIsHovered(false);
 
     return (
-        <div 
-            className="bg-white h-[17.6%] m-0"
-            onMouseEnter={onMouseEnter}
-            onMouseLeave={onMouseLeave}
-        >
+        <div onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
```

Comment:
An easier way thats built into tailwindcss is to have `hover:` prefix anything you want activated. Takes out the javascript from the equation and ur just using CSS-In-JS

Reply:
I think I had an instance of using hover: in SB_ExpandButton.js, however when making test cases, I switched it to using these onMouseEnter and Leave states so fireEvent could pick up whether it was hovered over or not. It might be because I am an amateur at creating test cases but I couldn't figure out how to check for 'hover:bg-cyan-400' so I swapped to this in testing.